lib/mongodb-filters.js

```diff
@@ -70,7 +81,9 @@ module.exports = class MongoDBFilters {
 
 			const filterKey = this.getFilterFieldName(filterName, modelField);
 
-			const filterValue = this.getFilterValueObject(filterData, modelField);
+			const mapper = modelField?.mapper || this.defaultMappers[filterName];
```

Comment:
Una giladita, pero quizas cambiarlo asi, para poder sacar el filtro default de alguno de los campos?

```suggestion
			const mapper = modelField?.mapper ?? this.defaultMappers[filterName];
```